packages/protocol-kit/src/types/index.ts

```diff
@@ -2,7 +2,7 @@ import { SafeTransactionOptionalProps } from '@safe-global/protocol-kit/utils/tr
 import {
   EthAdapter,
   MetaTransactionData,
-  SafeContract,
+  SafeContactType,
```

Comment:
There is a typo here:
```suggestion
  SafeContractType,
```